python/fastsim/simdrive.py

```diff
@@ -948,14 +940,14 @@ def set_power_calcs(self, i):
 
             if self.veh.fc_eff_type == H2FC:
                 self.min_mc_kw_2help_fc[i] = max(
-                    self.trans_kw_in_ach[i], -self.cur_max_mech_mc_kw_in[i])
+                    self.trans_kw_in_ach[i], -self.ess_lim_mc_regen_kw[i])
```

Comment:
@calbaker Replacing this redundant variable makes the name kind of ambiguous, so I'd be cool with changing it to something more descriptive (for example, if we changed it back to `cur_max_mech_mc_kw_in` or something)

Reply:
can just use old name